src/fragments/lib/datastore/flutter/advanced-workflows/custom-primary-key/query-snippet.mdx

```diff
@@ -0,0 +1,41 @@
+Query by primary key value:
```

Comment:
@renebrandel - you asked that we have toggles between PK and PK + SK examples. Thoughts on that, here?

Reply:
Ideally we should keep them consistent across all platforms using tabs.

Reply:
@HuiSF ^

Reply:
Hi @renebrandel it looks like the tabs used on the delete example code snippets in iOS and Android doc is for providing multiple examples for different language or language specs, for Dart code example we don't need these.

Reply:
@HuiSF if you look at the JS docs, you’ll see that we use code switchers for PK / composite key samples. This makes the docs shorter / easier to read (since all code switchers will be toggled in unison).